org.freedownloadmanager.Manager.yaml

```diff
@@ -0,0 +1,78 @@
+app-id: org.freedownloadmanager.Manager
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: '21.08'
+    add-ld-path: .
+command: fdm
+tags:
+  - proprietary
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --filesystem=xdg-download
+
+modules:
```

Comment:
You should add `"cleanup": ["/include"]`.

Reply:
Done!